robot/description/urdf/stand_link.urdf.xacro

```diff
@@ -7,7 +7,34 @@
     <!-- Stand link; this is a dummy link that is used to make camera placement and other sensors on the stand more convenient -->
     <!-- Stand link definition. Note: purposely left blank -->
     <link name="${tf_prefix}/stand_link">
+    </link> 
+
+    <!-- ZED Camera -->
+    <link name="${tf_prefix}/zed_camera_link">
+        <visual>
+            <origin xyz="0 0 0" rpy="0 0 0"/>
+            <!-- TODO: should be using the mesh but couldn't get it to work -->
+            <geometry>
+            <box size="${ZED_camera_width} ${ZED_camera_length} ${ZED_camera_height}"/>
+            </geometry>
+        </visual>  
+        <collision>
+            <origin xyz="0 0 0" rpy="0 0 0"/>
+            <geometry>
+                <box size="${ZED_camera_width} ${ZED_camera_length} ${ZED_camera_height}"/>
+            </geometry>
+        </collision>
+        <inertial>
+            <xacro:box_inertia mass="${ZED_camera_mass}" length="${ZED_camera_length}" width="${ZED_camera_width}" height="${ZED_camera_height}"/>
+            <mass value="${ZED_camera_mass}"/>
+        </inertial>
     </link>
+    <joint name="${tf_prefix}/zed_camera_joint" type="fixed">
+        <!-- magic number to be fixed: 1.42 -->
+        <origin xyz="${stand_length/2} 0 ${stand_height + ZED_camera_height - 1.42}" rpy="0 0 0"/>
+        <parent link="${tf_prefix}/stand_link"/>
+        <child link="${tf_prefix}/zed_camera_link"/>
```

Comment:
**Note:** you are connecting the camera w/r to `stand_link`, which is defined as the center, on top of the "stand plate". So you don't need any `z` offset when placing you camera (`z = 0`). Note that 1.42 is not really a magic number, you are using it to cancel out the `stand_height` term - *also* if the origin of the camera is at it's center, then u want to use `z = ZED_camera_height / 2.0`)